test/spec/unit/core/adapterManager_spec.js

```diff
@@ -1142,5 +1144,85 @@ describe('adapterManager tests', function () {
         expect(bidRequests[0].gdprConsent).to.be.undefined;
       });
     });
+
+    describe('s2sTesting - testServerOnly', () => {
+      beforeEach(() => {
+        config.setConfig({ s2sConfig: getServerTestingConfig(CONFIG) });
+      });
+
+      afterEach(() => config.resetConfig());
+
+      const makeBidRequests = ads => {
+        let bidRequests = adapterManager.makeBidRequests(
+          ads, 1111, 2222, 1000
+        );
+
+        bidRequests.sort((a, b) => {
+          if (a.bidderCode < b.bidderCode) return -1;
+          if (a.bidderCode > b.bidderCode) return 1;
+          return 0;
+        });
+
+        return bidRequests;
+      };
+
+      it('should only call the server once for Rubicon', () => {
+        const bidRequests = makeBidRequests(getServerTestingsAds());
+
+        expect(bidRequests).lengthOf(1);
+
+        expect(bidRequests[0].bids).lengthOf(1);
+        expect(bidRequests[0].bids[0].bidder).equals('rubicon');
+        expect(bidRequests[0].bids[0].finalSource).equals('server');
+      });
+
+      it('should only call the server twice', () => {
```

Comment:
these test descriptions aren't very meaningful.  We should be describing an attribute of the `testServerOnly1` feature (from the `describe` clause).  For example:
1. it suppresses all client bids if there are `server` bids resulting from `bidSource` at the adUnit Level
2. it suppresses all client bids if there are `server` bids resulting from `s2sTesting.bidderControl`
3. it suppresses all client bids if there are `server` bids resulting from `bidSource` at the adUnit Level or `s2sTesting.bidderControl`
4. does not suppress any client bids if there are no `server` bids resulting from `bidSource` at the adUnit Level or `s2sTesting.bidderControl`
4. does not suppress client requests when it's `falsey`